src/test/java/com/opencart/tests/currency/ChangeCurrencyTest.java

```diff
@@ -33,13 +33,6 @@ public void getCurrenciesValuesFromAdminPage() {
             "for currencies that are presented on admin page")
     public void ChangeCurrencyTest() {
```

Comment:
The last comment, if you have only one test in class, you do not need to move precondition logic to before class, so please, put it into the test